app/build.gradle

```diff
@@ -22,9 +22,12 @@ android {
 
     lintOptions {
         warning 'MissingTranslation'
-        warning 'InvalidPackage' // https://github.com/square/okio/issues/58
+        disable 'InvalidPackage' // http://jakewharton.github.io/butterknife/
```

Comment:
The link is the Butterknife documentation. I can however revert the change and add the Butterknife docs as second reference.